packages/quicktype-core/src/language/CSharp.ts

```diff
@@ -346,7 +453,10 @@ export class CSharpRenderer extends ConvenienceRenderer {
     }
 
     protected namerForObjectProperty(): Namer {
-        return namingFunction;
+        if (this._csOptions.keepPropertyName)
```

Comment:
If you're omitting `{` and `}`, please just do this as a ternary.

Reply:
> If you're omitting `{` and `}`, please just do this as a ternary.

Thanks for your review, edited